rules/windows/discovery_adfind_command_activity.toml

```diff
@@ -0,0 +1,65 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the Active Directory query tool, AdFind.exe. AdFind has legitimate purposes, but it is frequently
+leveraged by threat actors to perform post-exploitation Active Directory reconnaissance. The AdFind tool has been
+observed in Trickbot, Ryuk, Maze, and FIN6 campaigns.
+"""
+index = ["logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "AdFind Command Activity"
+note = "`AdFind.exe` is a legitimate domain query tool. Rule alerts should be investigated to identify if the user has a role that would explain using this tool and that it is being run from an expected directory and endpoint. Leverage the exception workflow in the Kibana Security App or Elasticsearch API to tune this rule to your environment."
+references = [
+    "http://www.joeware.net/freetools/tools/adfind/",
+    "https://thedfirreport.com/2020/05/08/adfind-recon/",
+    "https://www.fireeye.com/blog/threat-research/2020/05/tactics-techniques-procedures-associated-with-maze-ransomware-incidents.html",
+    "https://www.cybereason.com/blog/dropping-anchor-from-a-trickbot-infection-to-the-discovery-of-the-anchor-malware",
+    "https://www.fireeye.com/blog/threat-research/2019/04/pick-six-intercepting-a-fin6-intrusion.html",
+    "https://usa.visa.com/dam/VCOM/global/support-legal/documents/fin6-cybercrime-group-expands-threat-To-ecommerce-merchants.pdf",
+]
+risk_score = 21
+rule_id = "eda499b8-a073-4e35-9733-22ec71f57f3a"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery"]
+type = "query"
+
+query = '''
+event.category:process and event.action:start and process.pe.original_file_name:AdFind.exe
```

Comment:
would need to verify casing too

```suggestion
event.category:process and event.action:start and (process.name:AdFind.exe or process.pe.original_file_name:AdFind.exe)
```

Reply:
@brokensound77 in order for this to be detected with Winlogbeat, it appears that Sysmon is needed, I couldn't get a detection without it. Sysmon doesn't appear to use `event.action:start`, it uses `event.action:"Process Create (rule: ProcessCreate)"`. I can call this out in the description, but I just wanted to check that I'm not missing something.

```
event.category:process and event.action:(start or "Process Create (rule: ProcessCreate)") 
and (process.name:AdFind.exe or process.pe.original_file_name:AdFind.exe)
```

![image](https://user-images.githubusercontent.com/7442091/100660901-4bddf080-3318-11eb-959c-ff269748ae2c.png)
![image](https://user-images.githubusercontent.com/7442091/100660858-3a94e400-3318-11eb-97b9-32f2891c2779.png)

Reply:
The most consistent categorical fields are `event.category` and `event.type` and `event.action` should only be used to refine it further. I modified my suggestion accordingly